code/modules/clothing/head/helmet.dm

```diff
@@ -560,7 +560,7 @@
 		return
 	attached_light.on = !attached_light.on
 	attached_light.update_brightness()
-	to_chat(user, "<span class='notice'>You toggle the helmet light [attached_light.on ? "on":"off"].</span>")
+	balloon_alert(user, "light toggled [attached_light.on ? "on":"off"]")
```

Comment:
Do we really need this? A big light comes from your head.

Reply:
you know what, flashlights don't use feedback, so neither will this.